### ISSUE TITLE
Kindly provide support to generate code in Java Using Rest Assured

### ISSUE THREAD

**Issue by joinsaad** [postmanlabs/postman-code-generators#202]
There is support for Java UniRest and Java OkHttp, but there is no support for Java Rest Assured which is widely used java library to test Rest APIs

**Comment by sastava007** [comment]
@joinsaad Can you please share some stats in support of your request?

**Comment by joinsaad** [comment]
Rest Assured is constantly featured as one of the top tools/libraries when automating APIs. 
Here are some links 
Listed at number 1 here in open source API automation tools.
https://techbeacon.com/app-dev-testing/11-top-open-source-api-testing-tools-what-your-team-needs-know

Number 5 according to Joe Colnatonio
https://testguild.com/12-open-source-api-testing-tools-rest-soap-services/

Listed at number 7 in the list of all commercial and open-source API automation tools.
https://medium.com/@alicealdaine/top-10-api-testing-tools-rest-soap-services-5395cb03cfa9

In short, whenever Postman is listed on any list, the Rest-Assured is also present there. Even though its a java library but the ease of writing code in that is amazing and much better than okHttp and UniRest.

**Comment by aamol** [comment]
Is there any update on when this feature can be available ?
